### PR TITLE
Add community packages page

### PR DESCRIPTION
**Pull request by PragmaTwice**
After:
image

### CODE REVIEW

**Review by torwig — APPROVED** [review]

File: download/community-packages.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Community Packages
```
Comment (tisonkun) [inline download/community-packages.md:1]: Let's call it "Third-party" instead of "Community" because Kvrocks is also a community and can be ambiguous.

Comment (PragmaTwice) [inline download/community-packages.md:1]:
```suggestion
# Third-party Packages
```

Comment (PragmaTwice) [reply download/community-packages.md]: done

File: download/community-packages.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Third-party Packages
+
+In addition to [official releases](https://kvrocks.apache.org/download/), 
+there are some packages for different linux distros maintained by some individual community members.
+
+Please note that these packages are NOT created and sponsored by the ASF.
```
Comment (PragmaTwice) [inline download/community-packages.md:6]:
```suggestion
Please note that these packages are NOT created or sponsored by the ASF.
```

**Review by tisonkun — COMMENTED** [review]
I'm pushing a commit to update the file name also.

**Review by tisonkun — APPROVED** [review]

**Comment by PragmaTwice** [comment]
@tisonkun Thank you!

**Pull request merged**